_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -45,24 +45,54 @@ In order to use the `s3` source, configure your AWS Identity and Access Manageme
 
 If your S3 objects or Amazon SQS queues do not use [AWS Key Management Service (AWS KMS)](https://aws.amazon.com/kms/), remove the `kms:Decrypt` permission.
 
+## Cross-account S3 access
+
+### Bucket ownership<a name="s3_bucket_ownership"></a>
+
+When Data Prepper gets data from the S3 bucket, it verifies the ownership of the bucket using the 
+[bucket owner condition](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-owner-condition.html).
+By default, Data Prepper will expect that the S3 bucket is owned by the same account that owns the SQS queue.
+If not SQS queue is provided, it will use the role ARN in the `aws` configuration.
+
+If you are using cross-account S3 access, you need to configure how Data Prepper checks this condition.
+
+If all of your buckets are in one other account, you can specify the `default_bucket_owner` and set it to account Id of that account owning the other buckets.
+If your buckets are in multiple accounts, you should use the `bucket_owners` map.
+
+In this example, the SQS queue is owned by account `000000000000`. It will have data from two S3 buckets: `my-bucket-01` and `my-bucket-02`.
+These buckets are owned by different accounts - `my-bucket-01` is owned by `123456789012` and `my-bucket-02` is owned by `999999999999`.
```

Comment:
```suggestion
Since `my-bucket-01` is owned by `123456789012` and `my-bucket-02` is owned by `999999999999`, the `bucket_owners` map calls both bucket owners with their account IDs, as shown in the following configuration:
```